src/Table/Column/Checkbox.php

```diff
@@ -50,13 +50,16 @@ public function getHeaderCellHtml(Field $field = null, $value = null): string
                 ->addMoreInfo('field', $field);
         }
         $this->table->js(true)->find('.' . $this->class)->checkbox();
+        $this->table->js(true, new JsExpression('atk.gridCheckboxHelper.masterCheckbox();'));
 
-        return parent::getHeaderCellHtml($field);
+        return $this->getTag('head', [['div', ['class' => 'ui master fitted checkbox ' . $this->class], [['input/', ['type' => 'checkbox']]]]], ['class' => ['collapsing']]);
     }
 
     #[\Override]
     public function getDataCellTemplate(Field $field = null): string
     {
-        return $this->getApp()->getTag('div', ['class' => 'ui checkbox ' . $this->class], [['input/', ['type' => 'checkbox']]]);
+        $this->table->js(true, new JsExpression('atk.gridCheckboxHelper.childCheckbox();'));
```

Comment:
this must be moved into `getHeader...` otherwise it will be added multiple times because of #2146